li-hadoop-plugin/src/main/resources/LiBangBangJob.template

```diff
@@ -0,0 +1,21 @@
+/**
```

Comment:
I think we should actually add the Apache license template as a comment at the top of this file